partitions.go

```diff
@@ -231,13 +231,17 @@ func checkBootEnvAndRootPartitionMatch(bootPartNum string, rootPart string) bool
 	return strings.HasSuffix(rootPart, bootPartNum)
 }
 
-func resolveLink(path string) string {
+func resolveLink(unresolvedPath string) string {
```

Comment:
I like the variable rename!

I would also rename `resolveLink` to `maybeResolveLink` or something similar, just so that future readers know to look up the function and not blindly assume that it always resolves links.